_includes/example/in-page-config.html

```diff
@@ -1,3 +1,4 @@
+<meta property="laterpay:connector:article_id" content="custom_article_id_for_in_page_config_example_page">
```

Comment:
The [docs](http://docs.laterpay.net/platform/dialogs/add/#parameters) say about `article_id`:

> An unique, service consumer assigned identifier for the current article. Can be any UTF-8 string (☃ [Unicode Snowman] is not allowed). This is subsequently used in calls to /access.

And in more detail [here](http://docs.laterpay.net/platform/backend/main_endpoints/#get-access):

> Some unique identifier for the current article. Article ids have no significance/meaning to LaterPay, their format and structure is up to you, and they do not require any form of “pre-registration”. Must be a UTF-8 string (except ☃ [Unicode Snowman]).